pallets/nft/src/tests.rs

```diff
@@ -27,7 +27,7 @@ fn mint_nft_works() {
 		// TODO: DispatchError for ClassNotFound
 		// assert_noop!(
 		//     Nft::mint_nft(Origin::signed(ALICE), vec![0], pending_nft.clone()),
-		//     crate::Error::<Runtime>::NftError(orml_nft::Error::<Runtime>::ClassNotFound)
+		//     crate::Error::<Runtime>::NftError(base_nft::Error::<Runtime>::ClassNotFound)
```

Comment:
Do we need the code in this comment later? I replaced the text just in case, but if not maybe we just delete it.

Reply:
A good practice is to never leave blocks of code commented. If needed we can always revert back.
CC: @korzewski

Reply:
I'll get rid of it